docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+  - Explore configuration files for VmAnomaly and VmAlert.
+  - Run our own VictoriaMetrics database with data scraped from NodeExporter.
+  - Explore data for analysis in Grafana.
+  - Explore VmAnomaly results.
+  - Explore VmAlert alerts
+
+ 
+
+
+_____________________________
+
+## Data to analyze
+Let's talk about data used for anomaly detection in this tutorial. 
+We are going to collect our own CPU usage data with node-exporter into the VictoriaMertics database.
+
+On a Node exporters' metrics page, part of the output looks like this:
+```
+# HELP node_cpu_seconds_total Seconds the CPUs spent in each mode.
+# TYPE node_cpu_seconds_total counter
+node_cpu_seconds_total{cpu="0",mode="idle"} 94965.14
+node_cpu_seconds_total{cpu="0",mode="iowait"} 51.25
+node_cpu_seconds_total{cpu="0",mode="irq"} 0
+node_cpu_seconds_total{cpu="0",mode="nice"} 0
+node_cpu_seconds_total{cpu="0",mode="softirq"} 1682.18
+node_cpu_seconds_total{cpu="0",mode="steal"} 0
+node_cpu_seconds_total{cpu="0",mode="system"} 995.37
+node_cpu_seconds_total{cpu="0",mode="user"} 12378.05
+node_cpu_seconds_total{cpu="1",mode="idle"} 94386.53
+node_cpu_seconds_total{cpu="1",mode="iowait"} 51.22
+node_cpu_seconds_total{cpu="1",mode="irq"} 0
+node_cpu_seconds_total{cpu="1",mode="nice"} 0
+node_cpu_seconds_total{cpu="1",mode="softirq"} 39.18
+node_cpu_seconds_total{cpu="1",mode="steal"} 0
+node_cpu_seconds_total{cpu="1",mode="system"} 1378.79
+node_cpu_seconds_total{cpu="1",mode="user"} 14241.59
+node_cpu_seconds_total{cpu="2",mode="idle"} 93929.1
+node_cpu_seconds_total{cpu="2",mode="iowait"} 50.93
+node_cpu_seconds_total{cpu="2",mode="irq"} 0
+node_cpu_seconds_total{cpu="2",mode="nice"} 0
+node_cpu_seconds_total{cpu="2",mode="softirq"} 30.41
+node_cpu_seconds_total{cpu="2",mode="steal"} 0
+node_cpu_seconds_total{cpu="2",mode="system"} 1368.52
+node_cpu_seconds_total{cpu="2",mode="user"} 14722.17
+node_cpu_seconds_total{cpu="3",mode="idle"} 93554.49
+```
+Here, `node_cpu_seconds_total` metric tells us how many seconds each CPU spent doing each type of work: _user_, _system_, _iowait_, _idle_, _irq&softirq_, _guest_, or _steal_.
+These modes are mutually exclusive. A high _iowait_ means that you are disk or network bound, high user or system means that you are CPU bound.
+
+We'll send this node_exporter data into VictoriaMetrics later in this tutorial. But lets just note that `node_cpu_seconds_total` metric is a growing counter, so to calculate the per-second values we will use the rate function: `rate(node_cpu_seconds_total)`.
+Here is how this query may look like in Grafana:
+![node_cpu_rate_graph](guide-vmanomaly-node-cpu-rate-graph.png "node_cpu_rate_graph")
+
+This query result will generate 8 data vectors per each cpu, and we will use them as an input for our VM Anomaly Detection. VmAnomaly will start learning configured model type separately for each of the data vectors.
```

Comment:
it is better to stick to [time series ](https://docs.victoriametrics.com/keyConcepts.html#time-series)definiton instead of data vectors
you also have 18 series on the screenshot, not 8.

Reply:
8 per each cpu label. So 40 in total.
Grafana just doesn't show all the labels.